app/src/main/res/layout-v17/fragment_tool_details.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.ccci.gto.android.common.widget.HackyNestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fillViewport="true">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <androidx.percentlayout.widget.PercentRelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <org.ccci.gto.android.common.picasso.view.SimplePicassoImageView
+                android:id="@+id/banner"
+                android:adjustViewBounds="true"
+                android:background="@color/gray_E6"
+                android:scaleType="centerCrop"
+                app:layout_aspectRatio="210%"
+                app:layout_widthPercent="100%" />
+
+            <com.pierfrancescosoffritti.androidyoutubeplayer.core.player.views.YouTubePlayerView
```

Comment:
instead of duplicating the entire layout I believe you should be able to use an `<import />` tag with a layout containing `<merge />` for v16 or a layout with the player view for v17+

Reply:
@frett can we talk in person I am not 100% clear on what you are referring to.